File: angular/src/app/doctor/models/address-dto.ts

import { CityDto } from './city-dto';

import { EntityDto } from '@abp/ng.core';

export class AddressDto extends EntityDto<number> {
  streetName: string;
  districtName: string;
  postalcode: number;
  buildNumber: number;
  apartmentNumber: number;
  cityId: number;
  city: any;
  id: number;

  constructor(initialValues: Partial<AddressDto> = {}) {
    super(initialValues);
  }
}

File: aspnet-core/src/ERP.Healthcare.Application.Contracts/Doctors/AddressDto.cs
using Volo.Abp.Application.Dtos;

namespace ERP.Healthcare.Doctors
{
    public class AddressDto : EntityDto<int>
    {
        public string StreetName { get; set; }

        public string DistrictName { get; set; }

        public int Postalcode { get; set; }

        public int BuildNumber { get; set; }

        public int ApartmentNumber { get; set; }

        public int CityId { get; set; }

        public CityDto City { get; set; }
    }
}
